packages/renderer/src/lib/image/RunImage.svelte

```diff
@@ -386,10 +386,7 @@ function checkContainerName(event: any) {
 
 <Route path="/*" let:meta>
   {#if dataReady}
-    <NavPage
-      title="Create a container from image {imageDisplayName}"
-      searchEnabled="{false}"
-      subtitle="{image.tag}@{image.shortId} ">
```

Comment:
Maybe this can be useful to be shown somewhere else in the title?
I only see this otherwise
![immagine](https://user-images.githubusercontent.com/49404737/224700054-d47f6e2e-7960-4fd3-951d-c9b1587949c6.png)

Reply:
You'd prefer something like this?

<img width="743" alt="Screenshot 2023-03-13 at 8 24 28 AM" src="https://user-images.githubusercontent.com/19958075/224701129-b5568325-c391-4bfa-a3b7-b1b2bbc80e94.png">

I didn't think the sha was helpful, maybe just imageDisplayName:image.tag?

Reply:
yeah i was referring to the tag. I am not sure about the sha. It is something i don't look at honestly but maybe it is useful in some cases 🤷‍♂️

Reply:
I added a commit to add the tag to the title, using ":". IMHO this is standard and helpful, but I don't think the SHA is useful enough (at least not in title/subtitle).

<img width="568" alt="Screenshot 2023-03-13 at 9 01 31 AM" src="https://user-images.githubusercontent.com/19958075/224709314-cf912c61-037d-454b-b43f-0aca127e6fb1.png">